### ISSUE TITLE
Can't pass a string to n[] anymore without downgrading cypher engine

### ISSUE THREAD

**Issue by tomzeppenfeldt** [neo4j/neo4j#11764]
Version 3.3.5.

In the statement below , you have to downgrade the cypher, otherwise, you get an error when the property `keyname` does not exist.

Neo.DatabaseError.Schema.PropertyKeyAccessFailed: Property key 'No such property.' not found

```
//cypher 3.2 
with 'keyname' as keyname
match (n) 
with keyname,n limit 1
return n[keyname]
```

Obviously, it should return `null` in case the property does not exist

**Comment by SaschaPeukert** [comment]
Thanks for reporting. I could reproduce it and this seems like a bug to me. I will let you know when it's fixed.

Best regards
Sascha, Neo4j Cypher Team

**Comment by eduelias** [comment]
Is there a prevision to release this fix?

**Comment by SaschaPeukert** [comment]
The fix should be in the now availible 3.3.6
